api/envoy/api/v3alpha/core/backoff.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package envoy.api.v3alpha.core;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/versioning.proto";
+
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.api.v3alpha.core";
+option java_outer_classname = "BackoffProto";
+option java_multiple_files = true;
+
+// [#protodoc-title: Backoff]
+
+// Configuration defining a jittered exponential back off strategy.
+message BackOffStrategy {
+  option (udpa.annotations.versioning).previous_message_type = "envoy.api.v2.core.BackOffStrategy";
+
+  // The base interval to be used for the next back off computation. It should be greater than
+  // zero and less than or equal to :ref:`max_interval
+  // <envoy_api_field_BackOffStrategy.max_interval>`.
+  google.protobuf.Duration base_interval = 1 [(validate.rules).duration = {
+    required: true
+    gt {}
```

Comment:
I think `gt {}` implies `required`, since it implies a non-empty message. But I might be mistaken, can you check?